### DISCUSSION TITLE
Shelly EM (ADE7953) not showing negative power

### DISCUSSION THREAD

**Discussion by revk** [arendst/Tasmota#15362] [Q&A] [answered]
I have tried several searches and all I can see is a release note comment mentioning negative values as "fixed".
I have tasmota 11.1.0 on a Shelly EM, and it is only showing +ve values for power, even when exporting.
Is there some setting I am missing that allows it to correctly show -ve. It seems the ADE7953 can show this.
Thanks.

**Comment by sfromis** [comment, marked as answer]
Already asked: https://github.com/arendst/Tasmota/discussions/15315 https://github.com/arendst/Tasmota/discussions/12861

**Reply by revk** [reply]
> Already asked: https://github.com/arendst/Tasmota/discussions/15315 https://github.com/arendst/Tasmota/discussions/12861

Thanks, so stupid question, which release/version has that change in it so I can upgrade?

**Reply by revk** [reply]
OK, it looks like the "fix" for negative values was merged in 2021. https://github.com/arendst/Tasmota/pull/12874

Great, so any clues why it does not actually work? I do not get negative values.

`{"NAME":"Shelly EM","GPIO":[0,0,0,0,0,0,0,0,640,3456,608,224,0,1],"FLAG":0,"BASE":18}`

**Reply by revk** [reply]
Ok the template was a "1" next to ADE7953_IRQ, changing to a "2" seems to mark it as a Shelly EM, rather than a Shelly 2.5, and that fixes it. Of course it was not obvious as, at the time I changed it, I was exporting power, *AND*, it turns out I had the clamp backwards, so did not change. But all good now. Thank you all.

**Reply by AlfaBravoX** [reply]
had same problem. thanks for sharing the solution
